lib/menu.rb

```diff
@@ -0,0 +1,18 @@
+class Menu
+
+
+  attr_reader :foods
+
+  def initialize
+    @foods = { "pizza" => 7.99,
+              "pasta" => 7.50,
+              "steak" => 11.99,
+              "pint" => 3.99 }
+  end
```

Comment:
Your list of dishes would most likely be coming from some sort of external file if this were a real world program. Therefore pulling it in as @ljones140 suggests would reduce this code smell considerably.